pdr_backend/pdr_dashboard/pages/home.py

```diff
@@ -0,0 +1,158 @@
+from dash import html, dcc
+import dash_bootstrap_components as dbc
+from pdr_backend.pdr_dashboard.dash_components.view_elements import (
+    get_table,
+    get_feeds_data,
+    get_predictoors_data,
+    get_metric,
+    get_date_period_selection_component,
+)
+from pdr_backend.pdr_dashboard.dash_components.util import (
+    get_feed_ids_based_on_predictoors_from_db,
+)
+
+
+def layout(app):
+    return html.Div(
+        [
+            dcc.Store(id="user-payout-stats"),
+            dcc.Loading(
+                id="loading",
+                type="default",
+                children=get_main_container(app),
+                custom_spinner=html.H2(dbc.Spinner(), style={"height": "100%"}),
+            ),
+        ],
+        style={"height": "100%"},
+    )
+
+
+def get_main_container(app):
+    return html.Div(
+        [get_input_column(app), get_graphs_column()],
+        className="main-container",
+    )
+
+
+def get_graphs_column():
+    return html.Div(
+        [get_graphs_column_metrics_row(), get_graphs_column_plots_row()],
+        id="graphs_container",
+        style={
+            "height": "100%",
+            "width": "70%",
+            "display": "flex",
+            "flexDirection": "column",
+            "justifyContent": "start",
+            "paddingLeft": "50px",
+        },
+    )
+
+
+def get_input_column(app):
+    feed_cols, feed_data = get_feeds_data(app)
+    predictoor_cols, predictoor_data = get_predictoors_data(app)
+
+    selected_predictoors = list(range(len(app.favourite_addresses)))
+
+    if app.favourite_addresses:
```

Comment:
I know this is my work, but seeing it moved helped me realise we can simplify the ifs a bit. If we move app as a class attribute and it's easier to separate, let's retrieve the selected feeds in a separate function and keep the functional part away from the layout part. i.e. have a function "get_feeds_for_predictoors" instead of this block here (58-71)